install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```diff
@@ -10,4 +10,5 @@ spec:
     - name: http
       port: 80
       targetPort: 12000
-  type: ClusterIP
+  type: {{ .Values.global.hubble.ui.serviceType | default "ClusterIP" }}
```

Comment:
If I am not wrong, we want to keep global config as minimal as possible. I feel like this param should be part of hubble-ui chart only (e.g. .Values.serviceType). 

Another point is to make it extensible, maybe introduce service object, and user service.nodePort here ?

Reply:
Thanks, I didn't know about such goals.
Didn't work a lot with subcharts. Will try to fix this 👍

Reply:
@sayboras 
I didn't work with subcharts before.

If I understand correctly this should look like this...
in hubble-ui chart template: `.Values.service.serviceType`
helm install would look like: 
```
helm upgrade cilium cilium/cilium --version 1.8.2 \
            --namespace $CILIUM_NAMESPACE \
            ...
            --set hubble-ui.service.serviceType=NodePort
```

Reply:
yes, your above snippet looks good :+1:, you can check with `helm template` command as well

<details>
<summary>helm template</summary>

```
# default value with ClusterIP
helm template cilium ./install/kubernetes/cilium \
   --set global.hubble.ui.enabled=true \
   --set global.hubble.relay.enabled=true \
   --set global.hubble.enabled=true

# Custom serviceType value as NodePort
helm template cilium ./install/kubernetes/cilium \
   --set global.hubble.ui.enabled=true \
   --set global.hubble.relay.enabled=true \
   --set global.hubble.enabled=true \
   --set hubble-ui.service.serviceType=NodePort
```

<details>

Reply:
@sayboras  please take a look again